src/Client/app/(root)/my-exams/start-exam/page.tsx

```
@@ -54,93 +85,95 @@ export default function StartExam() {
                                     {currentExam?.title}
                                     <span
-                                        className={`ml-2 text-sm text-red-500`}
+                                        className={`ml-2 text-sm font-normal text-red-500`}
                                     >
                                         {currentExam?.status}
                                     </span>
                                 </h1>
-                                <div className="flex items-center gap-1 before:content-[''] before:w-2 before:h-2 before:bg-red-500 before:rounded-full">
-                                    <p className="min-w-[100px] text-sm">
+                                <div className="flex items-center gap-2">
+                                    <div className="w-2 h-2 rounded-full bg-red-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)